readme.md

```diff
@@ -0,0 +1,69 @@
+# Frontend Cookbook
+
+## Cookbooks
+
+* [React](/cookbook-react)
+* [React Native](/cookbook-react-native)
+* [Web](/cookbook-web)
+
+## Getting started
```

Comment:
I'd rename this to `Running locally` or something like that and move it to the bottom.